.github/workflows/build-push-dev-image.yml

```diff
@@ -1,105 +1,27 @@
 # Workflow responsible for the 
 # development release processes.
-#
 
 name: Build-Push-Dev-Image
 on:
-  push:
-    branches:
-      - develop
+   push:
+    # branches:
+    #   - develop
     paths-ignore:
       - README.md
-      - .old_cicd/*
-      - .github/*
-      - .github/workflows/*
+    #   - .old_cicd/*
```

Comment:
Uncomment these protections that keep this from running when actions are updated please. .